### ISSUE TITLE
No award for District Championship Points Qualifying Team

### ISSUE THREAD

**Issue by gregmarra** [the-blue-alliance/the-blue-alliance#1431]
/tasks/get/fmsapi_awards/2016inpmh, but also every event.

`Found an award without an associated type: District Championship Points Qualifying Team`

**Comment by fangeugene** [comment]
I added the award, but then I reverted the change because the award pages ended up looking super crowded. I decided this "award" wasn't really necessary because a) it's not a physical award teams get and b) you can see if a team makes it to district championships just by looking at their events. Is that reasonable?

![image](https://cloud.githubusercontent.com/assets/1421884/14066527/b7a9c214-f419-11e5-9d69-d033a6b7fb03.png)

**Comment by gregmarra** [comment]
Yes that sounds good, but we should make it so we don't log errors :)
